apps/web/src/pages/integrations/utils.ts

```diff
@@ -1,10 +1,24 @@
+import capitalize from 'lodash.capitalize';
 import { IEnvironment, providers } from '@novu/shared';
 
 import { IntegrationEntity } from './IntegrationsStorePage';
 import { CONTEXT_PATH } from '../../config';
 import { CHANNEL_TYPE_TO_STRING } from '../../utils/channels';
 import type { ITableIntegration } from './types';
 
+const makeNovuProviderName = (providerId: string) => {
+  return `${providerId}`
+    .split('-')
+    .map((el) => {
+      if (el.toLocaleLowerCase() === 'sms') {
+        return el.toUpperCase();
+      }
+
+      return capitalize(el);
+    })
+    .join(' ');
+};
```

Comment:
the Novu providers do not exist in the `providers` array tho there is no `displayName` for them... I feel like we shouldn't include them there because they "don't exist"... that's why this hacky function is needed which takes the `providerId` ex. `novu-sms` and makes out of it a readable provider name